src/DefaultTestReporter.js

```diff
@@ -62,6 +62,10 @@ function(config, testResult, aggregatedResults) {
     testRunTimeString = this._formatMsg(testRunTimeString, FAIL_COLOR);
   }
 
+  var resultHeader = this._getResultHeader(allTestsPassed, pathStr, [
+      testRunTimeString
+    ]);
```

Comment:
indent is messed here

(we need to have some code reformatter for a single code standard, but for now have to manually review these nits)